### ISSUE TITLE
Text from notes field display in Training Progress tag

### ISSUE THREAD

**Issue by maneesha** [swcarpentry/amy#1200]
See this example:
https://amy.software-carpentry.org/workshops/person/10629/

The notes from the Training Progress homework section show up in the tag itself.  This should still just say "SWC Homework" like it does when there are no notes.

**Comment by maneesha** [comment]
This does not seem to happen consistently.  See
https://amy.software-carpentry.org/workshops/person/7083/

**Comment by chrismedrela** [comment]
@maneesha I don't understand. Could you provide an example expected text for two cases: 1) when notes are present and 2) there are no notes. The current behaviour is:
1.  Notes are present:   
    ```
    Passed SWC Homework
    evaluated by Maneesha Sane
    on Monday 19 June 2017 at 14:30
    Notes: fake notes.
    Click to edit or delete.
    ```

2. No notes:
    ```
    Passed SWC Homework
    evaluated by Maneesha Sane
    on Monday 19 June 2017 at 14:30
    Click to edit or delete.
    ```

**Comment by pbanaszkiewicz** [comment]
This was fixed in c28de968845f98c6590784df1fe5beff7b3d021e.
